.circleci/config.yml

```diff
@@ -7,7 +7,7 @@ parameters:
     default: ""
   remote-deps:
     type: string
-    default: ""
+    default: cran/ggplot2@3.4.4
```

Comment:
Why is this needed?

Reply:
It was causing the splineforsimulatanousconfidenceintervals function to fail. ggplot2 is frozen on nightly, so this is just making it consistent. https://github.com/Displayr/r-server/blob/ea585a9518340aaa283484e131fc2ec2f41c8b3f/etc/r-packages-nightly#L41